submodules/rest_api/parse_api_pseudo_fs.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+
+import argparse
+import json
+import os
+import stat
+import sys
+
+from collections import defaultdict
+
+sys.path.append(os.getenv("MAIN_IMAGE_ENV_SHARED_LOCATION_ENV", ""))
+import read_api_fs_args
+import filesystem_utils
+from api_gen_utils import content_type_from_file_extension
+
+# import filesystem_utils
+# import api_gen_utils
+
+
+parser = argparse.ArgumentParser(
+    prog="Traverse across file-system API nodes and restore API from it"
+)
+
+parser.add_argument("mount_point", help="An existing mounted pseudo fs root node path")
+parser.add_argument("domain_name_api_entry", help="build API queries processor for that particular domain")
```

Comment:
mount_point consist of domain_name_api_entry as basename. Need to remove it from mount_point and use concatenation wtith domain manually